source/hassio/installation.markdown

```diff
@@ -36,7 +36,11 @@ The following will take you through the steps required to install Home Assistant
 2. Install Home Assistant:
 
    - Flash the downloaded image to an SD card using [balenaEtcher][balenaEtcher]. If using a Pi, we recommend at least a 32 GB SD card to avoid running out of space.
-   - Load the appliance image into your virtual machine software. Allow at least 32 GB of disk space for the virtual machine. Choose 64-bit Linux and UEFI boot. For a KVM-based setup with `virt-manager`, set **Firmware** to `UEFI x86_64: /usr/share/ovmf/x64/OVMF_CODE.fd`.
+   - Load the appliance image into your virtual machine software. (Note: You are free to assign as much resources as you wish to the VM, please assign enough based on your add-on needs)
+     - For VirtualBox create a new virtual machine, Select "Other Linux (64Bit) Assign it atleast 2GB of Memory and "Use an existing virtual hard disk file", select the VDI file from above, afterwards edit the "Settings" of the VM and go "System" then Motherboard and Enable EFI, then "Network" "Adapter 1" Bridged and your adapter
+     - For Hyper-V create a new virtual machine, Select "Generation 2", Assign it atleast 2GB of Memory and select "Connection -> "Your Virtual Switch that is bridged", then "Use an existing virtual hard disk" and select the VHDX file from above, after creation go to "Settings" -> "Security" and deselect "Enable Secure Boot"
+     - For KVM create a new virtual machine in virt-manager, select "Import existing disk image", provide the path to the QCOW2 image above, choose "Generic Default" for the OS, assign atleast 2GB RAM and 1vCPU, check the box for "Customize configuration before install" and select your bridge under "Network Selection", then under customization select "Overview" -> "Firmware" -> "UEFI x86_64: /usr/share/ovmf/x64/OVMF_CODE.fd"
```

Comment:
on my system (fedora 32, likely to be the same on EL systems), it's `/usr/share/edk2/ovmf/OVMF_CODE.fd`. It's prefilled, and the other UEFI option is "secboot" which I assume to be for secure boot.

Reply:
On F32 with `virt-manager` 2.2.1 it's ` /usr/share/edk2/ovmf/OVMF_CODE.fd`. I can't remember where the `/usr/share/ovmf/x64/OVMF_CODE.fd` was coming from in the first place.

Reply:
Debian/Ubuntu have `/usr/share/OVMF/OVMF_CODE.fd`, but no `/usr/share/ovmf/x64/OVMF_CODE.fd`